cosmos-admin/README.md

```diff
@@ -35,6 +36,30 @@ If for any reason the data copying is interrupted (for instance, the communicati
 
 [Top](#top)
 
+###<a name="warnings"></a>`warnings.sh`
+This script has been designed for detecting certain scenarios the Cosmos administrator must be warned about. Specifically:
+
+* When a HDFS space is close to the quota limit. How much close it is depends on a configurable threshold.
+* When an account has no data within the HDFS space and certain time has last after the account creation. The interval check depends on a configurable value measured in days.
+
+When any of the above situations is detected, an email is sent to the Cosmos administrator.
+
+`warnings.sh` is parameterized by:
+
+*  Host running the MySQL server
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/b770ceabb195f8f79891fca3c3b62603de778da3